parser.json

```diff
@@ -18,71 +18,70 @@
     "min_phantom_version": "5.0.0",
     "python_version": "3",
     "latest_tested_versions": [
-        "Module pdfminer.six-20201018.tar.gz, October 25 2021",
-        "Module pdfminer-20140328-py2-none-any.whl, October 25 2021"
+        "Module pdfminer.six-20211012, January 12 2022"
     ],
     "app_wizard_version": "1.0.0",
     "pip_dependencies": {
-        "wheel": [
-            {
-                "module": "defusedxml",
-                "input_file": "wheels/defusedxml-0.7.1-py2.py3-none-any.whl"
-            },
-            {
-                "module": "pdfminer",
-                "input_file": "wheels/pdfminer-20140328-py2-none-any.whl"
-            }
-        ]
-    },
-    "pip3_dependencies": {
```

Comment:
Are these module changes done by running the `pre-commit run --all-files`?

Reply:
yes, these changes are by running the pre-commit hook. Can you please re-review it and merge?